proto/forwarding/forwarding_notification.proto

```diff
@@ -24,25 +24,28 @@ option go_package = "github.com/openconfig/lemming/dataplane/proto/forwarding";
 
 // Event enumerates different types of events that can be generated.
 enum Event {
-  PORT_EVENT = 1;
+  EVENT_UNSPECIFIED = 0;
+  EVENT_PORT = 1;
 }
 
 // EventDesc describes an event.
 message EventDesc {
-  optional Event event = 1;
-  optional uint64 sequence_number =
+  Event event = 1;
+  uint64 sequence_number =
       2;  // Monotonically increasing sequence number of event
-  extensions 100 to max;
+  oneof desc {
+    PortEventDesc port = 3;
+  }
```

Comment:
i did extension -> oneof for everything. we may want to be able add different event types
"_event" suffix seems redundant since every thing in the message must be an event

Reply:
Sounds good